File: Python/basic_playwright.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch(headless=True)
    context = browser.new_context()
    page = context.new_page()

    url = "https://example.com"  # replace with target URL
    page.goto(url, timeout=60000)
    html = page.content()

    with open("page.html", "w", encoding="utf-8") as f:
        f.write(html)

    browser.close()

File: NodeJS/basic_playwright.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch({ headless: true });
  const context = await browser.newContext();
  const page = await context.newPage();

  const url = 'https://example.com'; // replace with target URL
  await page.goto(url, { timeout: 60000 });

  const html = await page.content();
  const fs = require('fs');
  fs.writeFileSync('page.html', html, 'utf8');

  await browser.close();
})();
